clippy.toml

```diff
@@ -1,3 +1,5 @@
 allow-unwrap-in-tests = true
 allow-expect-in-tests = true
-allow-panic-in-tests = true
\ No newline at end of file
+allow-panic-in-tests = true
+large-error-threshold = 196
```

Comment:
this seems like just kicking the can down the road -> could we not fix it "properly" instead? which errors is clippy complaining about? could we box something?